.circleci/config.yml

```diff
@@ -241,6 +241,22 @@ jobs:
             LIST=$(echo $TESTFILES | awk '{ gsub(" ", ",") ; system( "echo "  $0) }')
             echo $LIST
             npm run protractor -- --specs $LIST
+  cleanup-test-data:
+    <<: *defaults
+    steps:
+      - setup
+      - start-server
+      - run: bundle exec rails tests:e2e_cleanup
+  waiter:
+    docker: 
+      - image: cimg/node:current
+    steps:
+      - run: |
+          while [[ $(curl --location --request GET "https://circleci.com/api/v2/workflow/$CIRCLE_WORKFLOW_ID/job" --header "Circle-Token: $CCI_TOKEN"| jq -r '.items[]|select(.name != "waiter")|.status' | grep -c "running") -gt 0 ]]
```

Comment:
question: How did you set `$CCI_TOKEN` in CircleCI? I though this would be a per-user token, but it seems to work for me even though I didn't set one.

Reply:
I put a token I generated in my account in the environment variables - this reminds me that we will need to put one in the prod contexts - adding to the engineering notes